ACID/acid_tbc.sql

```diff
@@ -18633,6 +18633,9 @@ INSERT INTO `creature_ai_scripts` (`id`,`creature_id`,`event_type`,`event_invers
 ('1715011','17150','7','0','100','0','0','0','0','0','22','0','0','0','0','0','0','0','0','0','0','0','Vir\'aani Arcanist - Set Phase to 0 on Evade'),
 -- Gan'arg Tinkerer
 ('1715101','17151','0','0','100','1','7500','7500','14000','16000','11','32003','1','0','0','0','0','0','0','0','0','0','Gan\'arg Tinkerer - Cast Power Burn'),
+-- Felguard Legionnaire
+('1715201','17152','0','0','100','1','3000','3000','10000','12000','11','15496','1','0','0','0','0','0','0','0','0','0','Felguard Legionnaire - Cast Cleave'),
+('1715202','17152','0','0','100','1','6000','6000','14000','16000','11','32015','1','0','0','0','0','0','0','0','0','0','Felguard Legionnaire - Cast Cutdown'),
```

Comment:
Might be a typo here? Cutdown is id 32009
http://www.wowhead.com/npc=17152/felguard-legionnaire#abilities